packages/cmd/src/help/abstract/AbstractHelpGenerator.ts

```diff
@@ -163,25 +163,28 @@ export abstract class AbstractHelpGenerator implements IHelpGenerator {
             return;
         }
         for (const option of this.mCommandDefinition.options) {
-            const group = option.group;
-            if (!this.groupToOption[group]) {
-                this.groupToOption[group] = [];
-            }
-            const caseSensitive = this.getCaseSensitiveFlagByOptionName(option.name);
-            const optionAndAliases = this.getOptionAndAliasesString(option, caseSensitive);
-            this.groupToOption[group].push(optionAndAliases);
+            if (!option.hidden) {
```

Comment:
If this were written as:
```typescript
for (const option of this.mCommandDefinition.options.filter(opt => !opt.hidden)) {
```
it would eliminate the extra level of indentation added here.

Curious what you think about this? It definitely doesn't need to be changed 🙂

Reply:
fix in last commit